pkg/gateway/model_build_targets.go

```diff
@@ -156,14 +179,8 @@ func (t *latticeTargetsModelBuildTask) buildLatticeTargets(ctx context.Context)
 						TargetIP: address.IP,
 						Port:     int64(port.Port),
 					}
-
-					for _, definedPort := range definedPorts {
-						if target.Port == definedPort || definedPort == undefinedPort {
-							// target port matches httproute
-							targetList = append(targetList, target)
-						} else {
-							glog.V(6).Infof("Port does not match the target - port:%d, definedPort:%d, target:%v", target.Port, definedPort, target)
-						}
```

Comment:
Could you write a comment here to say "endPoints' port should be corresponding k8sService's targetPort"

Reply:
Added a bit of explanation, hope it is clear now